src/discussions/discussions-home/DiscussionSidebar.test.jsx

```diff
@@ -100,8 +99,8 @@ describe('DiscussionSidebar', () => {
         },
       })]);
     renderComponent();
-    await act(async () => expect(await screen.findAllByText('Thread by other users')).toBeTruthy());
+    waitFor(async () => expect(await screen.findAllByText('Thread by other users')).toBeTruthy());
     expect(screen.queryByText('Thread by abc123')).not.toBeInTheDocument();
-    expect(container.querySelectorAll('.discussion-post')).toHaveLength(postCount);
+    waitFor(() => expect(container.querySelectorAll('.discussion-post')).toHaveLength(postCount));
```

Comment:
Using `waitFor` without `await` can cause tests to pass when they shouldn't. By `await`ing the `findAllByText` above this, this line can stay the same as it was before this PR.

```suggestion
    expect(container.querySelectorAll('.discussion-post')).toHaveLength(postCount);
```